nservicebus/messaging/message-correlation.md

```diff
@@ -7,14 +7,18 @@ related:
  - nservicebus/messaging/headers
 ---
 
-Message correlation is the act of connecting instances of request messages with instances of response messages. Since all messages have a `Message Id` the `Correlation Id` is just a reference back to the specific message instance that caused this message to be sent back as a reply.
+Message correlation connects instances of request messages with instances of response messages. Each message has a `Message Id` and the `Correlation Id` is a reference back to the specific message instance that caused this message to be sent back as a reply.
```

Comment:
> RelatedTo is a pointer to the message that caused a given message to be sent. 

> the Correlation Id is a reference back to the specific message instance that caused this message to be sent back as a reply.

I still don't get what's the difference. Is correlationId used only for reply messages and RelatedTo to all "preceding" messages, even if it's not a reply message?
Pointer/reference, at the end of the day isn't that all just message id?